### PR TITLE
Switch to use new tftrt api (TrtGraphConverter)

### PR DESCRIPTION
**Pull request by aaroey**

### CODE REVIEW

**Comment by aaroey** [comment]
Hi @pooyadavoodi, I verified that with synthetic data all precision modes work with this PR, just FYI.

**Comment by pooyadavoodi** [comment]
@aaroey Thanks for the PR!

File: tftrt/examples/image-classification/image_classification.py [hunk 1]
```diff
@@ -354,15 +356,15 @@ def benchmark_process(path):
...
+
     if mode == 'validation':
         return validation_process
     elif mode == 'benchmark':
         return benchmark_process
     else:
         raise ValueError("Mode must be either 'validation' or 'benchmark'")
 
-    
+
```
Comment (pooyadavoodi) [inline tftrt/examples/image-classification/image_classification.py]: Is this extra space needed here?

Comment (aaroey) [reply tftrt/examples/image-classification/image_classification.py]: Removed.

**Pull request merged**